### ISSUE TITLE
hub aus aber ic trotzdem sehen möglich?

### ISSUE THREAD

**Issue by mambo91** [modelleicher/FS19_simpleIC#18]
moin moin erstmal danke für deine tollen mods ....mich stört es ein wenig das ich immer den hub an machen muss um simleic zu sehen wäre es möglich das iwann zu ändern?

**Comment by modelleicher** [comment]
Nein, simpleIC nutzt renderText um die "X" anzuzeigen (weil dadurch genaues ausrichten und einfügen von Texturen, Objekten ins Modell usw. wegfällt). Dass das alles synchron und möglichst Bugfrei ist wird das in der draw() Funktion aufgerufen und die wiederum wird automatisch deaktiviert wenn man die Huds deaktiviert.

LG

**Comment by mambo91** [comment]
okay schade ich dachte es sei möglich weil es so im fendt vom Ahran Modding verbaut ist allerdings mit ein anderem ic....wäre es möglich einen tür sonud mit einzufügen ?auch hier das beispiel von Ahran Modding....
